lib/howitzer/exceptions.rb

```diff
@@ -3,8 +3,8 @@ module Howitzer
   CommunicationError = Class.new(StandardError)
   ParseError = Class.new(StandardError)
   InvalidApiKeyError = Class.new(StandardError)
-  LocatorNotDefinedError = Class.new(StandardError)
-  BadLocatorParamsError = Class.new(StandardError)
+  ElementNotDefinedError = Class.new(StandardError)
```

Comment:
it looks like we never use this exception ElementNotDefinedError, i think it can be removed, because we cannot check that element defined or not.